src/system/ThemeModule/Resources/views/Admin/var.html.twig

```diff
@@ -0,0 +1,18 @@
+{% form_theme form with [
+'ZikulaFormExtensionBundle:Form:bootstrap_3_zikula_admin_layout.html.twig',
+'ZikulaFormExtensionBundle:Form:form_div_layout.html.twig'
```

Comment:
with some work I believe so. I cannot remember but I don't think this is natively supported (in Forms). But I remember seeing some implementations of this online. It would require a similar override of the `form_div_layout.html.twig` template. I would prefer to not implement that here, but could be considered for later.